### DISCUSSION TITLE
Miui Gallery module

### DISCUSSION THREAD

**Discussion by Banty8844** [community/community#110047] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Sir in miui Gallery module I can't set any wallpaper to my screen, there is no any option ,how can I set this?

**Comment by ettaboyle** [comment]
Hi @Banty8844,

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
